en/02_Developer_Guides/03_Forms/00_Introduction.md

```diff
@@ -51,17 +47,17 @@ class PageController extends ContentController
     
     public function HelloForm()
     {
-        $fields = new FieldList(
+        $fields = FieldList::create(
             TextField::create('Name', 'Your Name')
         );
 
-        $actions = new FieldList(
+        $actions = FieldList:create(
             FormAction::create('doSayHello')->setTitle('Say hello')
         );
 
-        $required = new RequiredFields('Name');
+        $required = RequiredFields::create('Name');
 
-        $form = new Form($this, 'HelloForm', $fields, $actions, $required);
+        $form = Form::create($this, __FUNCTION__, $fields, $actions, $required);
```

Comment:
```suggestion
        $form = Form::create($this, __FUNCTION__, $fields, $actions, $validator);
```

Reply:
Out of scope for this pass. Doesn't matter what you call the variable.